cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -96,8 +96,8 @@
    * Set of Action available in this tool.
    */
   private enum Action {
-    UPGRADE("Upgrades CDAP to 3.2\n" +
-              "  The upgrade tool upgrades the following: \n" +
+    UPGRADE("Upgrades CDAP to " + ProjectInfo.getVersion().toString() +
```

Comment:
You have to call getVersion() once more, to get the string. Or implement the toString in Version.java.